azure-pipelines.yml

```diff
@@ -145,6 +145,32 @@ jobs:
       includePatterns:
         - "/sw/***"
 
+- job: cw310_sw_build
+  displayName: Build Software required for CW310 FPGA synthesis
+  dependsOn: lint
+  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
+  pool:
+    vmImage: ubuntu-18.04
+  steps:
+  - template: ci/install-package-dependencies.yml
+  - bash: |
+      set -x
+      sudo util/get-toolchain.py \
+        --install-dir="$TOOLCHAIN_PATH" \
+        --release-version="$TOOLCHAIN_VERSION" \
+        --update
+    displayName: Install toolchain
+  - bash: |
+      . util/build_consts.sh
+      ./meson_init.sh -A
+      ninja -C "$OBJ_DIR" sw/device/otp_img/otp_img_export_fpga_cw310
+      ninja -C "$OBJ_DIR" sw/device/lib/testing/test_rom/test_rom_export_fpga_cw310
```

Comment:
Nit: If you switch this to something like:
```
      ninja -C "$OBJ_DIR" \
        sw/device/otp_img/otp_img_export_fpga_cw310 \
        sw/device/lib/testing/test_rom/test_rom_export_fpga_cw310
```
you might expose a bit more parallelism.

Reply:
True, though this portion takes 16s in total ;) Could look at stripping down the dependency install at the beginning at that takes a few minutes.